lib/irb/init.rb

```diff
@@ -140,7 +140,7 @@ def IRB.parse_opts(argv: ::ARGV)
         when "1"
           $VERBOSE = false
         else
-          $VERBOSE = true
+          Warning[:deprecated] = $VERBOSE = true
```

Comment:
Don't you need to consider category?

```
$ ruby --help
  -W[level=2|:category]
                  set warning level; 0=silence, 1=medium, 2=verbose
```

Reply:
I don't think considering it is required for this commit. I'm not against it, but it's significantly more involved in terms of implementation, and I'm not attempting to replicate all aspects of the behavior of `ruby -W` in this commit, only trying to address what I see as the most important issue.  I'll leave it up to the `irb` maintainers whether replicating all behaviors of `ruby -w` and `ruby -W` for `irb -w` and and `irb -W` is worth the implementation cost.